crates/matrix-sdk/src/account.rs

```diff
@@ -623,4 +628,34 @@ impl Account {
         });
         Ok(self.client.send(request, None).await?)
     }
+
+    /// Set the given account data event.
```

Comment:
The docs here a bit, how should i put it, spartanic. As it stands it's a bit hard to figure out what `content` can be.

If I take a look at the docs for [GlobalAccountDataEventContent], I'm greeted with a blanket trait implementation. Following the docs to [EventContent] shows me that absolutely everything implements `EventContent`. Following the docs to [GlobalAccountDataEventType] isn't clear right away either.


[GlobalAccountDataEventContent]: https://docs.ruma.io/ruma/events/trait.GlobalAccountDataEventContent.html
[EventContent]: https://docs.ruma.io/ruma/events/trait.EventContent.html
[GlobalAccountDataEventType]: https://docs.ruma.io/ruma/events/enum.GlobalAccountDataEventType.html

One has to note that `GlobalAccountDataEventType` is an associated type for the `EventType` trait, then filter through the implementors of `EventType` which use that associated type.

Can we improve those docs a bit and/or add an example?

Reply:
Hm, I guess we could actually have a proper list of impls for all individual event content traits. Good point that it's currently pretty hard to find the matching types.

Reply:
I've created https://github.com/ruma/ruma/issues/1268 for that.